### PR TITLE
adding tower license

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**JIRA**
https://issues.redhat.com/browse/INTLY-4223

Related pr: https://github.com/RHCloudServices/integreatly_dev/pull/41

### CODE REVIEW

File: CREDENTIAL_CONFIG.yml [hunk 1]
```diff
@@ -84,4 +84,13 @@ tower_instance_list:
 dummy_tower_host: <CHANGEME>
 dummy_tower_verify_ssl: True
 dummy_tower_username: <CHANGEME>
-dummy_tower_password: <CHANGEME>
\ No newline at end of file
+dummy_tower_password: <CHANGEME>
+
+# tower_license
+# Note: `"eula_accpted": true` is required in your license json but is not included when sent.
```
Comment (pmccarthy) [inline CREDENTIAL_CONFIG.yml]: Small typo in `eula_accepted`

File: CREDENTIAL_CONFIG.yml [hunk 2]
```diff
@@ -84,4 +84,13 @@ tower_instance_list:
...
 dummy_tower_verify_ssl: True
 dummy_tower_username: <CHANGEME>
-dummy_tower_password: <CHANGEME>
\ No newline at end of file
+dummy_tower_password: <CHANGEME>
+
+# tower_license
+# Note: `"eula_accepted": true` is required in your license json but is not included when sent.
+# Replace <CHANGE_ME> with your license json but keep the `"eula_accepted": true` entry
+dummy_license: |
```
Comment (pmccarthy) [inline CREDENTIAL_CONFIG.yml]: Should this be `dummy_tower_license`? With `dummy` being the environment in this case

Comment (laurafitzgerald) [reply CREDENTIAL_CONFIG.yml]: Yes it should. Fixing now.

**Comment by pmccarthy** [comment]
Ran the bootstrap process but it doesn't appear to populate these new vars. I think there might be a couple of steps missing, see steps 2 & 3 of the "Adding new variables" section of the README: https://github.com/integr8ly/tower_dummy_credentials#adding-new-variables

**Review by pmccarthy — APPROVED** [review]
Re-ran bootstrap with latest changes and can confirm the license vars are in place now:

```
inventories/group_vars/all/dummy_tower_credentials_list.yml

---
dummy_tower_host: '<CHANGEME>'
dummy_tower_verify_ssl: 'True'
dummy_tower_username: '<CHANGEME>'
dummy_tower_password: !vault
|
  $ANSIBLE_VAULT;1.1;AES256
  37336433313366363235303537633766653464653730323162323338633931363365663637363838
  3066353831306233386236316236316562333539656338360a343839303665336439663430363636
  35643738343035653736363564333030376463396332663137323231336436626539386439343863
  3366313336663461320a336330376534306339306365616565396632303333396137323934393066
  6239
dummy_tower_license: !vault
|
  $ANSIBLE_VAULT;1.1;AES256
  37373338626535396165323261616335616465353737383732306333656431303566386165356465
  3064623037623465316437663938336431653964343338660a396563333330613865323735616537
  30323532356339323861666361376337326430353433383236353066363061376462323630363964
  3262393534623235370a653337336530636461303461323433633261343161646435643261316462
  31623831393839343763636137313039353639643634376166366363376464323838303138343238
  6562336265666466393532303037393633613664623537363238
```

Looks good 👍

**Comment by laurafitzgerald** [comment]
@pmccarthy would you mind merging this one for me? I don't seem to have merge access here.

**Pull request merged**